### DISCUSSION TITLE
Telegram bot setup

### DISCUSSION THREAD

**Discussion by 0a1b** [orangecoding/fredy#108] [Q&A]
Hej,

I am trying to set up a job with a telegram bot.
I tried it with two bots but i always get 
"This did not work :-( I've received the following error: undefined"

Anyone knows how to deal with that?

**Comment by orangecoding** [comment]
Hey. Would you mind checking the logs and post them to me?

**Reply by 0a1b** [reply]
That's odd. I tried today again and it worked...
I visited the link it put in and a day passed... these are the only two things i see that have been different...

**Reply by orangecoding** [reply]
Not sure what you mean. So does it work now or not?

**Comment by 0a1b** [comment]
It works now.
Thank you for asking!
